scripts/parse_anglerfish_results.py

```diff
@@ -35,7 +35,7 @@ def find_latest_flowcell_run(currentStep: Process) -> str:
 
 
 def find_latest_anglerfish_run(latest_flowcell_run_path: str) -> str:
-    anglerfish_query = f"{latest_flowcell_run_path}/*anglerfish_run*"
+    anglerfish_query = f"{latest_flowcell_run_path}/**/anglerfish_run*"
```

Comment:
I belive the double asterisk is sort of a special feature of some shells, have you tested this on the LIMS server?

Reply:
Tried it now

```
In [6]: p="repos/**/scripts"

In [7]: from glob import glob

In [8]: pwd
Out[8]: '/opt/gls/clarity/users/glsai'

In [9]: glob(p)
Out[9]: ['repos/scilifelab_epps/scripts']
```

Seems to work :)